src/renderer/components/ErrorView.tsx

```diff
@@ -41,31 +42,74 @@ export default function ErrorView(props: Props) {
                             <hr/>
                             <p>
-                                {`We're having trouble connecting to ${props.appName}. We'll continue to try and establish a connection.`}
+                                <FormattedMessage
+                                    id='renderer.components.errorView.havingTroubleConnecting'
+                                    defaultMessage={'We\'re having trouble connecting to {appName}. We\'ll continue to try and establish a connection.'}
+                                    values={{
+                                        appName: props.appName,
+                                    }}
+                                />
                                 <br/>
-                                {'If refreshing this page (Ctrl+R or Command+R) does not work please verify that:'}
+                                <FormattedMessage
+                                    id='renderer.components.errorView.refreshThenVerify'
+                                    defaultMessage='If refreshing this page (Ctrl+R or Command+R) does not work please verify that:'
+                                />
                             </p>
                             <ul className='ErrorView-bullets' >
-                                <li>{'Your computer is connected to the internet.'}</li>
-                                <li>{`The ${props.appName} URL `}
+                                <li>
+                                    <FormattedMessage
+                                        id='renderer.components.errorView.troubleshooting.computerIsConnected'
+                                        defaultMessage='Your computer is connected to the internet.'
+                                    />
+                                </li>
+                                <li>
+                                    <FormattedMessage
+                                        id='renderer.components.errorView.troubleshooting.urlIsCorrect.appName'
+                                        defaultMessage='The {appName} URL '
+                                        values={{
+                                            appName: props.appName,
+                                        }}
+                                    />
                                     <a
 
                                         onClick={props.handleLink}
                                         href='#'
                                     >
                                         {props.url}
-                                    </a>{' is correct.'}</li>
-                                <li>{'You can reach '}
+                                    </a>
+                                    <FormattedMessage
+                                        id='renderer.components.errorView.troubleshooting.urlIsCorrect.isCorrect'
+                                        defaultMessage=' is correct'
+                                    />
```

Comment:
We can't append translated strings like this sentence and the one below (see [my blog post](https://developers.mattermost.com/blog/common-i18n-mistakes/#mistake-4-concatenating-translated-strings)). We should use react-intl's [rich text formatting thing](https://developers.mattermost.com/blog/common-i18n-mistakes/#solution-4-rich-text-formatting-in-react-intl) to do this instead